powershell/public/Invoke-Maester.ps1

```diff
@@ -129,6 +129,14 @@ Function Invoke-Maester {
         # This is required when using application permissions.
         [string] $MailUserId,
 
+        # Optional. The Team Id of where the Teams Channel Message should be send to. e.g. 5ba5cbbb-675e-4a8d-a382-a52960a18b4d
+        # No message will be sent if this parameter is not provided.
+        [string[]] $TeamId,
+
+        # Optional. The Channel Id of where the Teams Channel Message should be send to. e.g. 19:4a95f7d8db4c4e7fae857bcebe0623e6@thread.tacv2
+        # No message will be sent if this parameter is not provided.
+        [string[]] $ChannelId,
```

Comment:
Let's call this TeamChanelId so we know it's related to the other Teams property.